3Dmol/parsers.js

```diff
@@ -2208,45 +2208,44 @@ $3Dmol.Parsers = (function() {
      */
     parsers.lammpstrj = parsers.LAMMPSTRJ = function(str, options){
  	var atoms = [];
-        var dic = {'id':'serial','xs':'x','ys':'y','zs':'z'};
+        var dic = {'id':'serial','type':'atom','xs':'x','ys':'y','zs':'z'};
         var lines = str.split(/\r?\n|\r/);
-        while (lines.length > 0){
-            if (lines.length < 9)
-                break;
-            var offset = 0;
-            var atomCount = 0;
-            for (var i=0; i<lines.length; i++){
-                if (lines[i].match(/ITEM: ATOMS/)){
-                    offset = i+1;
+        var offset = 0;
+        var atomCount = 0;
+        var start = 0;
+        while (start<lines.length-9){
```

Comment:
The box lines are always there regardless of whether the boundary conditions are fixed or periodic or shrink wrapped.